### ISSUE TITLE
v11: assign an access level using an administration rule doesn't work

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6524]
**Describe the bug**
If you create a source to assign an access level, it doesn't work.

**To Reproduce**
1. Create an Htpasswd source (or try to edit existing `file1` administration rule
2. Assign an access level of ALL using a catchall administration rule

=> Frontend displays following error message:

`config/sources: administration_rules.0.actions.0.value: Field must contain a single value`

**Expected behavior**
You should be allowed to do this action.

On top of that, we should be allowed to enter several values, not only one. So the error message is completely wrong.

**Additional context**
- [JSON payload sent by frontend ](https://gist.github.com/nqb/3db6349fc50a48f42fb912b59a9fd7e0)
- Reply from backend:
```
{
  "errors": [
    {
      "field": "administration_rules.0.actions.0.value",
      "message": "Field must contain a single value"
    }
  ],
  "message": "Unable to validate",
  "status": 422
}
```

**Comment by jrouzierinverse** [comment]
The payload sends ```administration_rules[0].actions[0].value``` as an array instead of a comma separated string.
@satkunas was this a change of behavior in what the frontend sends?

**Comment by jrouzierinverse** [comment]
Fixed by c24d165a8641f2b4b27abc4d74172687fc3e39dd
